### PR TITLE
trained network

### LINKED ISSUE #96: Accordingly, I think the b60 netwokr is not used to train
Accordingly, I think the b60 netwokr is not used to train

_Originally posted by @DouletMedia in https://github.com/changcheng967/Kata_speed/issues/95#issuecomment-2256549621_

### PR DESCRIPTION
**Pull request by changcheng967**
b20 network

### CODE REVIEW

**Comment by DouletMedia** [comment]
Did kata_speed gone

**Comment by changcheng967** [comment]
Yes accordingly

**Comment by DouletMedia** [comment]
> Yes accordingly

holy

**Comment by DouletMedia** [comment]
then there is not good

File: gatekeepersgf/stdout.txt [hunk 1]
```diff
@@ -13520,3 +13520,144 @@ winLossUtilityFactor = 1.0
 2024-07-29 13:13:57-0400: Rejecting kata1-b18c384nbt-s9937771520-d4300882049.bin.gz automatically since older than best accepted model
```
Comment (DouletMedia) [inline gatekeepersgf/stdout.txt:13520]: Can it not reject

File: gatekeepersgf/stdout.txt [hunk 2]
```diff
@@ -13520,3 +13520,144 @@ winLossUtilityFactor = 1.0
...
+winLossUtilityFactor = 1.0
+
+2024-07-29 13:33:36-0400: Gatekeeper Engine starting...
+2024-07-29 13:33:36-0400: Git revision: 7939e2c3eb43df04e08de8c166889c6d5e937bfa
+2024-07-29 13:33:36-0400: Required candidate win prop: 0.5
+2024-07-29 13:33:36-0400: Loaded all config stuff, watching for new neural nets in C:/Users/chang/OneDrive/Documents/GitHub/Kata_speed/modelstobetested/
+2024-07-29 13:33:36-0400: Found new candidate neural net kata1-b18c384nbt-s9937771520-d4300882049.bin.gz
+2024-07-29 13:33:36-0400: Rejecting kata1-b18c384nbt-s9937771520-d4300882049.bin.gz automatically since older than best accepted model
+2024-07-29 13:33:36-0400: Moving C:\Users\chang\OneDrive\Documents\GitHub\Kata_speed\modelstobetested\kata1-b18c384nbt-s9937771520-d4300882049.bin.gz to C:/Users/chang/OneDrive/Documents/GitHub/Kata_speed/rejectedmodels/kata1-b18c384nbt-s9937771520-d4300882049.bin.gz
+2024-07-29 13:33:38-0400: All cleaned up, quitting
```
Comment (DouletMedia) [inline gatekeepersgf/stdout.txt:13663]: Did you exit or something

File: selfplay/kata1-b28c512nbt-s7168446720-d4316919285.bin.gz/selfplay-3BBE8E94D978B473.cfg [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+# Most of the remaining parameters here below apply regardless of the initialization, although a few of them
+# vary depending on handicap vs normal game, and some are explicitly disabled (e.g. initGamesWithPolicy on hint positions).
+
+initGamesWithPolicy = true  # Play the first few moves of a game high-temperaturely from policy
+policyInitAreaProp = 0.04 # The avg number of moves to play
+compensateAfterPolicyInitProb = 0.2 # Additionally make komi fair this often after the high-temperature moves.
+sidePositionProb = 0.020  # With this probability, train on refuting bad alternative moves.
+
+cheapSearchProb = 0.75  # Do cheap searches with this probaiblity
+cheapSearchVisits = 100  # Number of visits for cheap search
```
Comment (DouletMedia) [inline selfplay/kata1-b28c512nbt-s7168446720-d4316919285.bin.gz/selfplay-3BBE8E94D978B473.cfg:61]: could be lower

**Review by DouletMedia — CHANGES_REQUESTED** [review]
That's all

File: gatekeepersgf/stdout.txt (hunk 1 above)
Comment (changcheng967) [reply gatekeepersgf/stdout.txt:13520]: No it just do it like that

Comment (changcheng967) [reply gatekeepersgf/stdout.txt:13520]: like KataGo way

File: gatekeepersgf/stdout.txt (hunk 2 above)
Comment (changcheng967) [reply gatekeepersgf/stdout.txt:13663]: yeah, take too much time

File: selfplay/kata1-b28c512nbt-s7168446720-d4316919285.bin.gz/selfplay-3BBE8E94D978B473.cfg (hunk 3 above)
Comment (changcheng967) [reply selfplay/kata1-b28c512nbt-s7168446720-d4316919285.bin.gz/selfplay-3BBE8E94D978B473.cfg:61]: fine

File: gatekeepersgf/stdout.txt (hunk 2 above)
Comment (DouletMedia) [reply gatekeepersgf/stdout.txt:13663]: ok

**Review by DouletMedia — APPROVED** [review]

**Review by DouletMedia — APPROVED** [review]
good

**Comment by DouletMedia** [comment]
Accordingly, I think the b60 netwokr is not used to train

**Comment by changcheng967** [comment]
> Accordingly, I think the b60 netwokr is not used to train

yeah

**Review by DouletMedia — CHANGES_REQUESTED** [review]
change it to accept b60

**Review by DouletMedia — APPROVED** [review]
good to go

**Pull request merged**